ivy/functional/frontends/tensorflow/tensor.py

```diff
@@ -53,6 +53,9 @@ def shape(self):
     def get_shape(self):
         return tf_frontend.raw_ops.Shape(input=self)
 
+    def floordiv(self, x, y, name="floorviv"):
+        return tf_frontend.raw_ops.FloorDiv(x, y, name)
+
```

Comment:
Why are we using the `raw_ops` `FloorDiv` function here and not the frontend `tf.floordiv` which is defined here (https://github.com/unifyai/ivy/blob/0f33aebcbcd443174f27bb8b43f322853e4078a3/ivy/functional/frontends/tensorflow/math.py#L277)